woocommerce/woocommerce-gutenberg-products-block
# Fix nightly builds

## What
Fixes the nightly builds Github workflow. I've increased the node heap size limit to allow for the extra power the build script needs recently. I've created an issue (https://github.com/woocommerce/woocommerce/issues/42244) to monitor this over time so we can avoid this in the future, although I suspect we're running into problems here because we're generating a development build here, not production, which has a lot more bells and whistles.

## Why

The nightly build workflow is failing because the build process is using up too much memory (p1692008212605009-slack-C02UBB1EPEF)

## Testing Instructions



N/A

## Developer Testing Instructions

1. Go to https://github.com/woocommerce/woocommerce-blocks/actions/workflows/nightly-builds.yml
2. Click `Run Workflow` button.
3. Choose `fix/nightly-builds` as the branch
4. Click `Run Workflow`.
5. The workflow should complete successfully


## Screenshots or screencast



N/A

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


N/A
